images/Dockerfile.el5

```diff
@@ -0,0 +1,19 @@
+FROM zhenech/katello-client:el5
```

Comment:
should that come from my account, or from the official centos one, given you add the katello stuff later anyways?

Reply:
Good point - do you happen to have the Dockerfile for this?

Reply:
https://github.com/evgeni/docker-katello-client/blob/el5/Dockerfile

Reply:
plus all the crap from https://github.com/evgeni/docker-katello-client/tree/el5/repos because CentOS5 can burn in hell